Procfile

```diff
@@ -1,2 +1,2 @@
-web: python manage.py migrate && python manage.py runserver "0.0.0.0:$PORT"
```

Comment:
The "||" will result in if the "migrate" returns 0 the "runserver" won't happen at all.